packages/core/src/bus/Command.ts

```diff
@@ -0,0 +1,39 @@
+import { CommandDispatcher } from './CommandDispatcher';
+import { getTypeName } from '../utils';
+import { v4 as uuidv4 } from 'uuid';
+
+export abstract class Command<T, R> {
+  public readonly expectReply = true;
+  public readonly ttl = 10000;
```

Comment:
That is my fault https://github.com/NeuraLegion/secbox-sdk-js/pull/12#discussion_r839246888

We should leave type declaration, otherwise, it won't allow us to override them in the derived class

```suggestion
  public readonly expectReply: boolean = true;
  public readonly ttl: number = 10000;
```